### PR TITLE
Added sample HTML page for Duo enrollment.

### PR DESCRIPTION
**Pull request by nikolaystoilov-okta**
Resolves: OKTA-104925

Reviewers: 
@mystiberry-okta @tthompson-okta @srinivasanagandla-okta

### CODE REVIEW

File: _source/_docs/api/resources/authn.md [hunk 1]
```diff
@@ -1534,6 +1534,40 @@ curl -v -X POST \
 </script>
 ~~~
 
+#### Full Page Example for Duo enrollment
+In this example will will put all the elements together in html page. It is important to provide the stateToken information. Some additional information on how to integrate with Duo is provided [here](https://duo.com/docs/duoweb#passing-additional-post-arguments-with-the-signed-response).
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/authn.md]: will will > we will
in html page > in an HTML page

**Comment by mystiberry-okta** [comment]
@nikolaystoilov-okta can you resolve the conflicts? I'm afraid I'll make a mistake with your content if I do it. We're republishing the site early this afternoon, FYI.

**Comment by srinivasanagandla-okta** [comment]
I think the conflicts must be resolved before we review the changes. Can you resolve those?

**Comment by mystiberry-okta** [comment]
@srinivasanagandla-okta I'm afraid I'll make errors in your content if I do the resolve.

**Comment by mystiberry-okta** [comment]
check out the conflicts here: https://github.com/okta/okta.github.io/pull/538/conflicts  @srinivasanagandla-okta

File: _source/_docs/api/resources/authn.md [hunk 2]
```diff
@@ -1658,18 +1658,38 @@ curl -v -X POST \
...
+        </form>
+
+        <script src="https://<your org>.okta.com/js/sections/duo/Duo-Web-v2.js"></script>
+
+        <!-- The host, sig_request, and post_action values will be given via the Auth API -->
+        <script>
+            Duo.init({  
+                'host': '<your org duo host>.duosecurity.com',
+                'sig_request': '<activation signature>',
+                'post_action': 'https://<your org>okta.com/api/v1/authn/factors/dsf2lw7BzJeNJwBo10g4/lifecycle/duoCallback'
```
Comment (srinivasanagandla-okta) [inline _source/_docs/api/resources/authn.md]: Instead of hard coding the factorId in the url, we should put a place holder and provide a comment about it.

**Comment by srinivasanagandla-okta** [comment]
@nikolaystoilov-okta Do we need similar documentation related to HTML for verify as well?

**Pull request merged**